cmake/legacy.cmake

```diff
@@ -242,10 +242,10 @@ ExternalProject_Add(hepmc
 )
 
 list(APPEND packages vc)
-set(vc_version "1.4.3")
+set(vc_version "1.4.4")
 ExternalProject_Add(vc
   URL https://github.com/VcDevel/Vc/releases/download/${vc_version}/Vc-${vc_version}.tar.gz
```

Comment:
```suggestion
  URL https://github.com/VcDevel/Vc/archive/refs/tags/${vc_version}.zip
```
🤷‍♂️

Reply:
There's only the source code link for this version: https://github.com/VcDevel/Vc/releases/tag/1.4.4
Can we build from that?

Reply:
Seems to be the same thing though.

Reply:
I've tested this locally and it builds. I'll commit to see how it does on the CI. Can drop if anything is wrong.